samples/bluetooth/llpm/src/main.c

```diff
@@ -209,7 +214,11 @@ static void disconnected(struct bt_conn *conn, uint8_t reason)
 		default_conn = NULL;
 	}
 
-	advertise_and_scan();
+	if (conn_info.role == BT_CONN_ROLE_MASTER) {
```

Comment:
The `connected` callback requires the similar handling.

Reply:
Fixed